backend/event-parse/parse.go

```diff
@@ -117,10 +119,26 @@ func (n networkFetcher) fetchStylesheetData(href string) ([]byte, error) {
 		return nil, errors.Wrap(err, "error reading styles")
 	}
 
+	body = replaceRelativePaths(body, href)
 	body = append([]byte("/*highlight-inject*/\n"), body...)
+
 	return body, nil
 }
 
+func replaceRelativePaths(body []byte, href string) []byte {
+	const PROXY_URL string = "https://replay-cors-proxy.highlightrun.workers.dev"
+
+	u, err := url.Parse(href)
+
+	if err == nil {
+		base := u.Scheme + "://" + u.Host
+
+		return regexp.MustCompile(`url\(['"]\./`).ReplaceAll(body, []byte("url('"+PROXY_URL+"?url="+base+"/"))
```

Comment:
prefer `fmt.Sprintf("...", PROXY_URL, base)` instead of concatenating the strings with `[]byte("url('"+PROXY_URL+"?url="+base+"/")`

Reply:
Is this preferred?

```
return regexp.MustCompile(`url\(['"]\./`).ReplaceAll(body, []byte(fmt.Sprintf("url('%s?url=%s/", PROXY_URL, base)))
```

Reply:
yup! sorry i wasn't too clear